.linguirc

```diff
@@ -17,5 +18,13 @@
       ]
     }
   ],
-  "format": "po"
+  "formatOptions": {
+    "lineNumbers": false
+  },
+  "fallbackLocales": {
+    "default": "en"
+  },
+  "format": "po",
+  "orderBy": "messageId",
+  "pseudoLocale": "pseudo"
```

Comment:
could you tell what is pseudo?

Reply:
Pseudo is basically a way to text if the translation is implemented properly or not when not even a single translation is done. I was using this to test when the translation was not done. 

https://lingui.js.org/guides/pseudolocalization.html